std-bits/base/src/main/java/org/enso/base/read/BaseReadManyReturnSPI.java

```diff
@@ -0,0 +1,14 @@
+package org.enso.base.read;
+
+@org.openide.util.lookup.ServiceProvider(service = ReadManyReturnSPI.class)
+public class BaseReadManyReturnSPI extends ReadManyReturnSPI {
```

Comment:
Repeating the [terminology misuse](https://github.com/enso-org/enso/pull/10714#discussion_r1697927708) - the `BaeReadManyReturn` class isn't _interface_, this class is _implementation_. I suggest to rename to `BaseReadManyReturnImpl`.

Make `final`.

Reply:
All of our SPI implementations are named according to this convention.

If the convention is wrong, we should rename them all at once, which should be done separately from this PR.